browser/components/DataSelectorDialogue.js

```diff
@@ -35,13 +36,19 @@ function DataSelectorDialogue(props) {
                 <IconButton icon="no3-solid" label={__('Nitrat')} />
                 <IconButton icon="water-drop-wifi-solid" label={__('Mine stationer')} />
                 <IconButton icon="lab-flask-experiment" label={__('Mine favoritter')} />
-                <Grid container>
+                <Grid container spacing={32}>
```

Comment:
Could the spacing be derived from `theme.layout.gutter`

Reply:
From the Material-UI Grid documentation, the grid is based on the "8px" rule. So a spacing of 4 gives 32px.